analytics_data_api/v0/apps.py

```diff
@@ -0,0 +1,13 @@
+from django.apps import AppConfig
+from django.conf import settings
+from elasticsearch_dsl import connections
+
+
+class ApiAppConfig(AppConfig):
+
+    name = 'analytics_data_api.v0'
+
+    def ready(self):
+        super(ApiAppConfig, self).ready()
+        if settings.ELASTICSEARCH_LEARNERS_HOST:
+            connections.connections.create_connection(hosts=[settings.ELASTICSEARCH_LEARNERS_HOST])
```

Comment:
Good point!  We're mocking out the calls in our unit tests, so this shouldn't be hit.